tests/dvslib/dvs_vlan.py

```diff
@@ -0,0 +1,94 @@
+from dvs_database import DVSDatabase
+
+class DVSVlan(object):
+    def __init__(self, adb, cdb, sdb, cntrdb, appdb):
+        self.asic_db = adb
+        self.config_db = cdb
+        self.state_db = sdb
+        self.counters_db = cntrdb
+        self.app_db = appdb
+
+    def create_vlan(self, vlanID):
+        vlanName = "Vlan{}".format(vlanID)
+        vlan_entry = {"vlanid": vlanID}
+        self.config_db.create_entry("VLAN", vlanName, vlan_entry)
+
+    def remove_vlan(self, vlanID):
+        vlanName = "Vlan{}".format(vlanID)
+        self.config_db.delete_entry("VLAN", vlanName)
+
+    def create_vlan_member(self, vlan, interface, tagging_mode="untagged"):
+        member = "Vlan{}|{}".format(vlan, interface)
```

Comment:
Should we check interface exist in PORT table,  Or for test, it is better to provide caller an option to do it irrespective of interface exist in PORT table or not.

Reply:
No, it's the caller's responsibility. These functions are used by test cases.